DQMServices/Core/src/DQMStore.cc

```diff
@@ -405,14 +405,14 @@ void DQMStore::mergeAndResetMEsRunSummaryCache(uint32_t run,
 	{
 	  if(me->getTH1()->CanExtendAllAxes() && i->getTH1()->CanExtendAllAxes()) {
 	    TList list;
-	    list.Add(i->getTH1());
+	    list.Add(i->getTH1().operator->().operator->());
```

Comment:
The temporary could go away before `Add()` is called. I'd think one would have to do something like
```
auto t = i->GetTH1();
list.add(*(t.get()));
```